config.json.sample

```diff
@@ -0,0 +1,3 @@
+{
+    "featureManagerIsFullZoneProvisioningEnabled": false
+}
```

Comment:
Sure, lets put it in the README instead of the sample.  I don't want the sample merging duplicate keys every time if we can avoid it.